tools/codegen/gofilegen/schema/schema_file.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func GenerateGoCode(input *codespec.Resource, withObjTypes bool) []byte {
-	schemaAttrs := GenerateSchemaAttributes(input.Schema.Attributes)
+	schemaAttrs := GenerateSchemaAttributes(input)
 	models := GenerateTypedModels(input.Schema.Attributes, withObjTypes)
```

Comment:
```suggestion
	schemaAttrs := GenerateSchemaAttributes(input)
	models := GenerateTypedModels(schemaAttrs.Schema.Attributes, withObjTypes)
```
We are not leveraging generation of typed models but would adjust for consistency

Reply:
schemaAtrtrs is type CodeStatement so doesn't have Schema